### PR TITLE
Feature/118 119 122 update ug

### LINKED ISSUE #118: Document GFS input data
## Describe the Enhancement ##
Add more documentation to the UG (and website?) to clarify input data, what's available, etc. 
Consider adding limits to UI for date selection?

### Time Estimate ###
3 hours

### Sub-Issues ###
Consider breaking the enhancement down into sub-issues.
- [ ] *Add a checkbox for each sub-issue here.*

### Relevant Deadlines ###
*List relevant project deadlines here or state NONE.*

### Assignee ###
- [ ] Select **engineer(s)** or **no engineer** required
- [ ] Select **scientist(s)** or **no scientist** required

### Labels ###
- [ ] Select **component(s)**
- [ ] Select **priority**

### Projects and Milestone ###
- [ ] Select **Project**
- [ ] Select **Milestone** as the next official version or **Backlog of Development Ideas**

## Enhancement Checklist ##
- [ ] Complete the issue definition above, including the **Time Estimate** and **Funding Source**.
- [ ] Fork this repository or create a branch of **develop**.
Branch name: `feature_<Issue Number>/<Description>`
- [ ] Complete the development and test your changes.
- [ ] Add/update log messages for easier debugging.
- [ ] Add/update tests.
- [ ] Add/update documentation.
- [ ] Push local changes to GitHub.
- [ ] Submit a pull request to merge into **develop**.
Pull request: `feature <Issue Number> <Description>`
- [ ] Define the pull request metadata, as permissions allow.
Select: **Reviewer(s)**, **Project**, and **Development** issue
Select: **Milestone** as the next official version
- [ ] Iterate until the reviewer(s) accept and merge your changes.
- [ ] Delete your fork or branch.
- [ ] Close this issue.

### PR DESCRIPTION
**Pull request by fossell**
This PR updates documentation in the Users Guide, in particular the administration and user interface sections, with some minor updates elsewhere. 

This closes #118, #119, and #122.

## Expected Differences ##

- [x] Do these changes modify the system output in any way? **[ No]**</br>
Only documentation updates.

## Pull Request Testing ##

- [x] Describe testing already performed for these changes:</br>
This feature branch passed the GH documentation build check. The UG was built on Readthedocs and reviewed for errors.

- [x] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>
View the feature branch on Readthedocs or code mods in the PR and suggest changes.

- [x] Do these changes include sufficient documentation updates, ensuring that no errors or warnings exist in the build of the documentation? **[Yes ]**

- [x] Do these changes include sufficient testing updates? **[Yes or No]**
N/A

- [x] Will this PR result in changes to the test suite? **[No]**</br>
If **yes**, describe the new output and/or changes to the existing output:</br>

- [ ] Please complete this pull request review by **[Fill in date]**.</br>
Wednesday, March 29

## Pull Request Checklist ##
- [x] Review the source issue metadata (labels, project, and milestone).
- [x] Complete the PR definition above.
- [x] Ensure the PR title matches the feature or bugfix branch name.
- [x] Define the PR metadata, as permissions allow.
Select: **Reviewer(s)**
Select: **Project**
Select: **Milestone** as the version that will include these changes
Select: **Development** to link to the original development issue.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue and delete your feature or bugfix branch from GitHub.

### CODE REVIEW

File: docs/Users_Guide/administration.rst [hunk 1]
```diff
@@ -1,63 +1,115 @@
...
+
+Users with the **Admin** role have permission to manage the users on their system. Admin users will see the menu option "Manage Users" on their User Interface, which allows the admin to add new users, update active user roles, or delete users. Select the "Manage Users" menu option to view a current list of users. 
+
+**Add a New User**
+
+Select the "Manage Users" menu option and click on the "Add User" option at the top of the page. A popup box will appear to fill in the new user's email address and name. Select the Role for that user and click "Create". An email will be sent to the new user asking them to activate their account and setup a new password.
+
+**Update Roles**
+
+Select the "Manage Users" menu option to view a current list of users. Click on the user you wish to update. A popbox box with the user's information will appear. Select the new role you wish to assign and click "Update". The user will automatically be given permissions associated with this new role.
```
Comment (hahnd) [inline docs/Users_Guide/administration.rst:97]: "popbox box" -> "pop-up box"

File: docs/Users_Guide/administration.rst [hunk 2]
```diff
@@ -1,63 +1,115 @@
...
+
+Select the "Manage Users" menu option and click on the "Add User" option at the top of the page. A popup box will appear to fill in the new user's email address and name. Select the Role for that user and click "Create". An email will be sent to the new user asking them to activate their account and setup a new password.
+
+**Update Roles**
+
+Select the "Manage Users" menu option to view a current list of users. Click on the user you wish to update. A popbox box with the user's information will appear. Select the new role you wish to assign and click "Update". The user will automatically be given permissions associated with this new role. 
+
+**Delete a User**
+
+Select the "Manage Users" menu option to view a current list of users. Click on the user you wish to remove. A popbox box with the user's information will appear. Click "Remove" to delete the user from the system.
```
Comment (hahnd) [inline docs/Users_Guide/administration.rst:101]: "popbox box" -> "pop-up box"

File: docs/Users_Guide/administration.rst [hunk 3]
```diff
@@ -1,63 +1,115 @@
...
+.. _admin_manage_users:
+
+Managing Users
+--------------
+
+Users with the **Admin** role have permission to manage the users on their system. Admin users will see the menu option "Manage Users" on their User Interface, which allows the admin to add new users, update active user roles, or delete users. Select the "Manage Users" menu option to view a current list of users. 
+
+**Add a New User**
+
+Select the "Manage Users" menu option and click on the "Add User" option at the top of the page. A popup box will appear to fill in the new user's email address and name. Select the Role for that user and click "Create". An email will be sent to the new user asking them to activate their account and setup a new password.
```
Comment (hahnd) [inline docs/Users_Guide/administration.rst:93]: "popup box" -> "pop-up box"

File: docs/Users_Guide/graphics.rst [hunk 4]
```diff
@@ -7,7 +7,7 @@ Graphical Outputs
...
+The forecast plots can be accessed by clicking on the Job ID or "Open Viewer" button on that job from WRF Jobs menu option. This will open the Forecast Viewer for that job. Each available product will be listed in the right-hand menu and can be toggled on/off with the checkmark. Each product also has a transparency slider bar to adjust the field's transparency. Additionally, there is a Political Boundaries selector at the top of the products list that can be toggled on/off depending on preference. Automation play/pause/advance buttons and slider can be found at the bottom of the map. On the left, a vertical slider bar allows users to select different vertical pressure levels for appropriate products.
```
Comment (hahnd) [inline docs/Users_Guide/graphics.rst:10]: First sentence is a little confusing.  Suggestion:
The forecast plots can be accessed through the "WRF Jobs" menu option.  When the model has finished, an "Open Viewer" button will appear.  The button is also available in the pop-up box if you click on a job to see its details.

File: docs/Users_Guide/overview.rst [hunk 5]
```diff
@@ -60,11 +60,8 @@ Compute Cluster
...
+The weather forecasts use initial conditions from the Global Forecast System (GFS) and are pulled from AWS's S3 on demand for each forecast. The numrical weather prediction software used for making the forecasts is the Weather Research and Modeling version 4.4.1 (`WRF <https://www2.mmm.ucar.edu/wrf/users/>`_) system, including it's pre-processor WPS. The WRF model output is then post-processed using the `wrf-python <https://wrf-python.readthedocs.io/en/latest/index.html>`_ package version v1.3.2, and the Unified Post Processor version 4.0.1 (`UPP <https://dtcenter.org/community-code/unified-post-processor-upp>`_) and from there the data are processed for plotting and served up to the system's user interface and forecast viewer.
```
Comment (hahnd) [inline docs/Users_Guide/overview.rst:63]: Weather Research and Modeling -> Weather Research and Forecasting

**Review by hahnd — CHANGES_REQUESTED** [review]

File: docs/Users_Guide/administration.rst [hunk 6]
```diff
@@ -1,63 +1,115 @@
...
+* **admin:** Active user with permissions of a maintainer and also admin permissions to manage users and the system
 
 .. list-table:: Roles and Permissions
-   :widths: 10 10 10 10 10
+   :widths: 10 10 10 10 10 10
    :header-rows: 1
    
    * -   
      - admin
+     - maintainer
```
[the remark this answers is not in the document]
Comment (fossell) [reply docs/Users_Guide/administration.rst:37]: @hahnd - I added an X indicator for change password under the anonymous role. Is this sufficient or should we expand the permissions options to provide greater detail about each action possible? It seems a little much at this point, but I may be misunderstanding the roles.

[the remark this answers is not in the document]
Comment (hahnd) [reply docs/Users_Guide/administration.rst:37]: Change Password is different from Reset Password.  Maybe the distinction is too nuanced and will add confusion and we should just leave the anonymous column blank.

[the remark this answers is not in the document]
Comment (fossell) [reply docs/Users_Guide/administration.rst:37]: Ok, I edited it to be blank again. We can revisit this to see what level of detail makes most sense at a later time.

**Pull request merged**